Cynalytics/nl-kat-coordination
# Tasks contain property that tells the boefjerunner what network it is supposed to run on.

### Changes

Made it so that newly created tasks (made from an IPAddress OOI) contain an attribute that explains what network the task should be run on.

### Issue link

Closes https://github.com/minvws/nl-kat-coordination/issues/3222

### Demo

![image](https://github.com/user-attachments/assets/6ec57341-bd77-4da1-9b06-eff8c7c992ba)
Task not being ran because only boefje runners which check for the network: "internal-cynalytics" (which is currently not being active) has not picked it up.

### QA notes

(Note that a new environment variable has been made called "network_scopes")

Try creating a new IPAddress with network "internet" and observe that it runs successfully. And after that create another IPAddress with a new self-made network and observe that wont be ran.

When you run the boefje container with your new network name included in the environment variable then the task should be ran.



---

### Code Checklist







---

## Checklist for code reviewers:


---

## Checklist for QA:


### What works:

- _bullet point + screenshot (if useful) per tested functionality_

### What doesn't work:

- _bullet point + screenshot (if useful) per tested functionality_

### Bug or feature?:

- _bullet point + screenshot (if useful) if it is unclear whether something is a bug or an intended feature._
